src/sherpa_ai/database/user_usage_tracker.py

```diff
@@ -31,17 +35,21 @@ class UserUsageTracker:
     def __init__(
         self,
         db_name=cfg.DB_NAME,
-        max_daily_token=20000,
+        verbose_logger: BaseVerboseLogger = DummyVerboseLogger(),
     ):
         self.engine = create_engine(db_name)
         Session = sessionmaker(bind=self.engine)
         self.session = Session()
         self.create_table()
-        self.max_daily_token = int(max_daily_token)
+        self.max_daily_token = float(cfg.DAILY_TOKEN_LIMIT)
```

Comment:
Do the `float` type-casting in sherpa_ai.config so that the value is always trustworthy for all callers

Reply:
do this:
DAILY_TOKEN_LIMIT = float(environ.get("DAILY_TOKEN_LIMIT") or 20000.00)

instead of
DAILY_TOKEN_LIMIT = float(environ.get("DAILY_TOKEN_LIMIT")) or 20000.00